lab1/lab_1.ipynb

```diff
@@ -2648,10 +2604,10 @@
             "assert X_train.shape == (30877, 37)\n",
             "assert X_test.shape == (10293, 37)\n",
             "\n",
-            "assert X_train[:,0].min() == 0\n",
-            "assert X_train[:,0].max() == 1\n",
-            "assert -3 < X_train[:,26].min() < -2\n",
-            "assert 5 < X_train[:,26].max() < 6\n",
+            "assert X_train[:,26].min() == 0\n",
```

Comment:
skąd ta zmiana?

Reply:
Zacytuję treść zadania 

> 2. Stwórz `ColumnTransformer`, przetwarzający zmienne kategoryczne za pomocą `OneHotEncoder` (teraz już nie musimy robić `drop="first"`), a numeryczne za pomocą `StandardScaler`. Zaaplikuj go do odpowiednich kolumn.

Po zakodowaniu OneHot kolumna o indeksie 0 jest numeryczna (age), a ta w kolumnie 26 jest one hot, a po zastosowaniu standard scaler zmienna numeryczna nie będzie miała wartości maksymalnej = 1 i minimalnej = 0, lecz ustandaryzowane ze srednia 0, i odchyleniem standardowym 1. Za to zmienna one hot (kolumna 26) nie osiągnie podanych przedziałów, lecz będzie się składać z 0-1